openmdao/core/tests/test_problem.py

```diff
@@ -75,9 +75,13 @@ def test_feature_simple_run_once_compute_totals(self):
         prob.setup()
         prob.run_model()
 
-        assert_rel_error(self, prob['comp.f_xy'], -15.0)
+        totals = prob.compute_totals(of=['comp.f_xy'], wrt=['p1.x', 'p2.y'])
+        print("Total derivative of 'comp.f_xy' wrt to 'p1.x' is {}".format(totals[('comp.f_xy','p1.x')][0][0]))
+        print("Total derivative of 'comp.f_xy' wrt to 'p2.y' is {}".format(totals[('comp.f_xy','p2.y')][0][0]))
 
-        prob.compute_totals(of=['comp.f_xy'], wrt=['p1.x', 'p2.y'])
+        totals = prob.compute_totals(of=['comp.f_xy'], wrt=['p1.x', 'p2.y'], return_format= 'dict')
+        print("Total derivative of 'comp.f_xy' wrt to 'p1.x' is {}".format(totals['comp.f_xy']['p1.x'][0][0]))
+        print("Total derivative of 'comp.f_xy' wrt to 'p2.y' is {}".format(totals['comp.f_xy']['p2.y'][0][0]))
```

Comment:
shouldn't these be asserts? (which are converted to prints for the docs)?

Reply:
Good point.  If we are just printing out values, the test will never actually fail?

Reply:
I definitely got confused between what I wanted to see in the docs and what should be in the actual test.